### DISCUSSION TITLE
Scan custom lists

### DISCUSSION THREAD

**Discussion by huhuhu-guy** [DiegoFleitas/letterboxd-movie-justwatch#32] [Ideas]

**Comment by DiegoFleitas** [comment]
> Hi, I really like the site you built to check your letterboxd watchlist for streaming options. It would be great if you could add the option to paste the link to a letterboxd list (not a watchlist but a custom list) and have the list scanned for streaming options. Thanks for your contribution, I use this site a lot.

Hey, I'm glad you found my site useful!

I've thought about that feature too, and I think it's a good idea. It should be doable with some code changes. To be completely honest, I haven't touched this project for a while, but some maintenance is in order—especially because the platform it's hosted on (Fly.io) has changed their pricing plans for some of the resources it's using.

I'll try to look into it soon.
